packages/experimental/Shimmer/src/Shimmer.tsx

```diff
@@ -39,79 +38,65 @@ export const Shimmer = compose<ShimmerType>({
     );
 
     const startValue = useShimmerAnimation(memoizedShimmerData);
 
     return (rest: ShimmerProps) => {
-      const { uri, elements, ...mergedProps } = mergeProps(props, rest);
+      const { elements, ...mergedProps } = mergeProps(props, rest);
       const rows = [];
 
       if (elements) {
         for (let i = 0; i < elements.length; i++) {
           const element = elements[i];
-          if (element.type) {
-            if (element.type == 'rect') {
-              rows.push(
-                <Rect
-                  key={i}
-                  width={element.width}
-                  height={element.height}
-                  x={element.xPos}
-                  y={element.yPos}
-                  rx={element.borderRadius}
-                  ry={element.borderRadius}
-                />,
-              );
-            } else if (element.type == 'circle') {
-              rows.push(<Circle key={i} r={element.height / 2} cx={element.xPos} cy={element.yPos} />);
-            } else {
-              assertNever(element.type);
-            }
+          if (element.type == 'rect') {
+            rows.push(
+              <Rect
+                key={i}
+                width={element.width}
+                height={element.height}
+                x={element.x}
+                y={element.y}
+                rx={element.borderRadiusX}
+                ry={element.borderRadiusY}
+              />,
+            );
+          } else if (element.type == 'circle') {
+            rows.push(<Circle key={i} r={element.radius} cx={element.cx} cy={element.cy} />);
+          } else {
+            assertNever(element);
           }
         }
       }
+
       return (
         <Slots.root {...mergedProps}>
           <Defs>
             <AnimatedLinearGradient id="gradient" x1={startValue} y1={memoizedShimmerData.angle} x2="-1" y2="-1">
-              <Stop
-                offset="10%"
-                stopColor={uri ? memoizedShimmerData.gradientTintColor : memoizedShimmerData.shimmerTintColor}
-                stopOpacity={uri ? '0' : '1'}
-              />
-              <Stop offset="20%" stopColor={memoizedShimmerData.gradientTintColor} stopOpacity={memoizedShimmerData.gradientOpacity} />
-              <Stop
-                offset="30%"
-                stopColor={uri ? memoizedShimmerData.gradientTintColor : memoizedShimmerData.shimmerTintColor}
-                stopOpacity={uri ? '0' : '1'}
-              />
+              <Stop offset="10%" stopColor={memoizedShimmerData.shimmerColor} stopOpacity={1} />
+              <Stop offset="20%" stopColor={memoizedShimmerData.shimmerWaveColor} stopOpacity={memoizedShimmerData.gradientOpacity} />
+              <Stop offset="30%" stopColor={memoizedShimmerData.shimmerColor} stopOpacity={1} />
             </AnimatedLinearGradient>
+            <ClipPath id="shimmerView">{rows}</ClipPath>
```

Comment:
Was this meant to move into the Defs tag?

Reply:
I did it on purpose but to the best of my knowledge it doesn't matter for practical purposes on a ClipPath since it isn't rendered as a raw element, only as a clipRule.

It's specifically important that AnimatedLinearGradient is in Defs so that it isn't rendered on top of being used as a fill for the Rect (though for _those_ practical purposes the rect is unnecessary AFAIK, but not changing it atm).  Elements in defs aren't rendered as elements w/ <Use/> elements.